front/app/api/verification_methods/keys.ts

```diff
@@ -4,6 +4,16 @@ const baseKey = { type: 'verification_method' };
 
 const verificationMethodsKeys = {
   all: () => [baseKey],
+  list: () => [{ ...baseKey, operation: 'list', parameters: {} }],
+  item: ({ endpoint }: { endpoint: string }) => [
```

Comment:
This was another bug: the two endpoints (`first_enabled` and `first_enabled_verified_actions` were sharing the same cache key